src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  function deleteParam(params) {
```

Comment:
Consider moving the 'deleteParam' function outside of 'transformState' to improve readability and prevent unnecessary re-declarations.